Rakefile.rb

```diff
@@ -26,23 +27,25 @@ def json_write(file, json)
   # we only build any country that contains that string. For example:
   #    EP_COUNTRY_REFRESH=Latvia be rake countries.json
   to_build = ENV['EP_COUNTRY_REFRESH'] || 'data'
-
-  countries = @HOUSES.group_by { |h| h.split('/')[1] }.select do |_, hs|
-    hs.any? { |h| h.include? to_build }
+  countries = EveryPolitician.countries.select do |c|
+    c.slug.downcase.include? to_build.downcase
```

Comment:
Using `.include?` here means that setting `EP_COUNTRY_REFRESH=Dominica` will _also_ rebuild the Dominican Republic. That's not necessarily a bad thing, but it could be unexpected. Perhaps this should either do an exact match _or_ print out the countries it's refreshing?